### ISSUE TITLE
Unable to build and start docs server

### ISSUE THREAD

**Issue by GnaneshKunal** [RedisLabs/redislabs-docs#1354]
**Current behavior** (A short description of what you found)
Unable to start server

**Expected behavior** (A short description of what you expected to find)
Should start server

**Screenshots** (Paste or drag-and-drop a screenshot)
Gives this error:
```
Building sites … ERROR 2021/04/29 23:51:28 [en] REF_NOT_FOUND: Ref "/ri/" from page "rc/databases/monitor-performance.md": page not found
ERROR 2021/04/29 23:51:28 [en] REF_NOT_FOUND: Ref "/rc/subscriptions/" from page "rc/subscriptions/create-flexible-subscription.md": page not found
ERROR 2021/04/29 23:51:28 [en] REF_NOT_FOUND: Ref "/rc/subscriptions/" from page "rc/rc-quickstart.md": page not found
ERROR 2021/04/29 23:51:28 [en] REF_NOT_FOUND: Ref "/rc/databases/" from page "rc/rc-quickstart.md": page not found
ERROR 2021/04/29 23:51:30 [en] REF_NOT_FOUND: Ref "/rs/security/ldap/" from page "rs/security/ldap/cluster-based-ldap-authentication.md": page not found
ERROR 2021/04/29 23:51:30 [en] REF_NOT_FOUND: Ref "/rs/security/ldap/" from page "rs/security/ldap/cluster-based-ldap-authentication.md": page not found
ERROR 2021/04/29 23:51:30 [en] REF_NOT_FOUND: Ref "/rs/security/ldap/" from page "rs/security/ldap/cluster-based-ldap-authentication.md": page not found
ERROR 2021/04/29 23:51:30 [en] REF_NOT_FOUND: Ref "/rs/security/" from page "rs/security/ldap/enable-role-based-ldap.md": page not found
ERROR 2021/04/29 23:51:30 [en] REF_NOT_FOUND: Ref "/rs/security/" from page "rs/security/ldap/map-ldap-groups-to-roles.md": page not found
ERROR 2021/04/29 23:51:30 [en] REF_NOT_FOUND: Ref "/rs/security/ldap/" from page "rs/security/ldap/migrate-to-role-based-ldap.md": page not found

```

**Additional context** (Add any other information)

**Comment by lanceleonard** [comment]
The build won't occur if files cannot be found.  

Those missing files look to be fairly recent additions, so I would verify that you're up-to-date with master.  

(The project changes on a near daily basis and we've been reorganizing different parts of the project.)

**Comment by GnaneshKunal** [comment]
I'm up to date with master. I got the build failure in #1353 as well. And the PR was up to date with master. You can verify that with the parent commit.

**Comment by lanceleonard** [comment]
OK.  Respectfully: a quick look at your repo shows it's (as of this writing) four commits behind master.

I don't know that's precisely the problem you're running into, but I do know that these errors are reporting missing files.  For some reason, they're not being found.  Perhaps there's an uncommitted change getting in the way.  Perhaps a file's been added to your local `.gitignore`.  Perhaps it's something else.

But that's the problem.  The build process can't find those files.  I can confirm that when those files can be located, the build runs as intended (provided there aren't other environmental issues).

**Comment by GnaneshKunal** [comment]
> _ a quick look at your repo shows it is (as of this writing) four commits behind master._

Did pull master, still its giving the missing files' error.

Did a fresh pull and ran the build, still getting the error:

https://user-images.githubusercontent.com/20211167/116804712-a6d17900-ab3e-11eb-848d-a7871a2b48da.mp4

**Comment by lanceleonard** [comment]
Thanks for the updated instructions.  I've just followed the same steps on a new device (one that hasn't previously had the repo locally) and it compiled without error.

At this point, I would completely remove Hugo and any local copies of the repo.  I would also check the `git config` settings to make sure there isn't anything getting in the way.  Than I'd start over.

FWIW: As a matter of habit, I almost always do the following before trying to build:

``` bash
% git checkout master
% git pull
```

(When I ran my test, someone had updated the project between `git clone` and `hugo serve`.)
